src/metrics/listener.ts

```diff
@@ -99,6 +99,10 @@ export class MetricsListener {
     }
   }
 
+  public sendDistributionMetric(name: string, value: number, ...tags: string[]) {
+    this.sendDistributionMetricWithDate(name, value, new Date(Date.now()), ...tags);
```

Comment:
```suggestion
    this.sendDistributionMetricWithDate(name, value, new Date(), ...tags);
```

Reply:
The reason I am passing Date.now() inside new Date() constructor is because its easy to mock Date.now(). I can work on mocking Date() if you think its absurd to use Date.now() with in the new Date() constructor.

Reply:
Hmm, if that's the reason, then I think it's fine to leave it like that.